### PR TITLE
Update card page

### PR DESCRIPTION
**Pull request by jtsuero**

### CODE REVIEW

File: src/AppStore.js [hunk 1]
```diff
@@ -55,10 +59,10 @@ class AppStore {
 
   addCardToDeck = (deckId, cardId) => {
     this.decks[deckId].cardIds.push(this.cards[cardId]);
+    this.cards[cardId].decks.push(this.getDeckName(deckId));
```
Comment (mrap) [inline src/AppStore.js]: I think the `card.decks` should be tracked as deck ids vs deck names.

File: src/AppStore.js [hunk 2]
```diff
@@ -73,6 +77,10 @@ class AppStore {
     return deckNames;
   }
 
+  getDeckName = (deckId) => {
```
Comment (mrap) [inline src/AppStore.js]: Duplicate of line 31?

File: src/Card.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ class Card  {
       this.id = id;
       this.question = question;
       this.answer = answer;
+      this.decks = [];
```
Comment (mrap) [inline src/Card.js]: I think the `card.decks` should be tracked as deck ids vs deck names.

`card.deckIds`

File: src/Home.js [hunk 4]
```diff
@@ -1,12 +1,18 @@
...
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 
 function Home() {
 
   return(
+  <Router>
     <div>
-    Test Home page
+    <Switch>
```
Comment (mrap) [inline src/Home.js]: Indentation

File: src/AppStore.js (hunk 1 above)
Comment (jtsuero) [reply src/AppStore.js]: done

File: src/AppStore.js (hunk 2 above)
Comment (jtsuero) [reply src/AppStore.js]: removed.

**Pull request closed without merging**